packages/react-utilities/src/compose/types.ts

```diff
@@ -65,20 +65,20 @@ export type ComponentSlotProps<Component extends React.ComponentType> = Componen
  * The second param is an optional union of alternative types that can be specified for the `as` prop.
  *
  * ```
- * IntrinsicShorthandProps<'div'> // Slot is always div
- * IntrinsicShorthandProps<'button', 'a'> // Defaults to button, but allows as="a" with anchor-specific props
- * IntrinsicShorthandProps<'label', 'span' | 'div'>; // Defaults to label, but allows as="span" or as="div"
+ * IntrinsicSlotProps<'div'> // Slot is always div
+ * IntrinsicSlotProps<'button', 'a'> // Defaults to button, but allows as="a" with anchor-specific props
+ * IntrinsicSlotProps<'label', 'span' | 'div'>; // Defaults to label, but allows as="span" or as="div"
```

Comment:
There's a `{@link IntrinsicShorthandProps}` docs comment referencing this on line 42 that should likely be updated

Reply:
Done!